vendor/github.com/containerd/aufs/aufs.go

```diff
@@ -49,7 +49,7 @@ type snapshotter struct {
 
 // New creates a new snapshotter using aufs
 func New(root string) (snapshots.Snapshotter, error) {
```

Comment:
I'll make this change to `containerd/aufs` once this PR is ok to most people. :)